getting-started/jdbc/docker-compose-bootstrap-db.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+services:
+  polaris-bootstrap:
+    # IMPORTANT: the image MUST contain the Postgres JDBC driver and EclipseLink dependencies, see README for instructions
+    image: apache/polaris-admin-tool:postgres-latest
+    environment:
+      polaris.persistence.type: relational-jdbc
+      quarkus.datasource.db-kind: pgsql
+      quarkus.datasource.jdbc.url: jdbc:postgresql://postgres:5432/POLARIS
+      quarkus.datasource.username: postgres
+      quarkus.datasource.password: postgres
+    command:
+      - "bootstrap"
```

Comment:
Why not add this command to the "setup" section of the main `docker-compose.yml` file?

Reply:
The main `docker-compose.yml` file is still used for the Getting Started experience with Cloud DB's, which are ideally not bootstrapped more than once since it's not part of a Docker image. If we include the Bootstrap into that Docker file, it makes it impossible to reload other Docker services, like Spark or Trino since we'll get the message that a DB can only be bootstrapped once.

Reply:
good point - thx!